CHANGELOG.md

```diff
@@ -1,7 +1,41 @@
 # Changelog
 
+### 0.23.9 (2016-08-09)
```

Comment:
GitHub renders lists without the whitespace, but it's not "proper markdown", and some engines do not, so I think we should use whitespace. For this PR, I'd prefer not to touch the original commits, but I'll send a follow-up to clean up the file.